doc/manual/migrating.rst

```diff
@@ -0,0 +1,53 @@
+.. _migrating
+
+Migrating from EEGLAB
+=====================
+
+To read in data exported from EEGLAB, MNE offers an EDF reader :func:`mne.io.read_raw_edf` and a `set` file reader.
+To read in `set` files containing ``raw`` data, use :func:`mne.io.read_raw_eeglab` and to read in `set` files containing
+``epochs`` data, use :func:`mne.read_epochs_eeglab`.
+
+Here is a cheatsheet to help users migrate painlessly from EEGLAB. For the sake of clarity, let us assume
+that the following are already defined or known: the file name ``fname``, low-pass cut-off frequency ``lfreq``,
+time interval of the epochs ``tmin`` and ``tmax``, and the conditions ``cond1`` and ``cond2``.
+
++-------------------+--------------------------------------------------------------+-----------------------------------------------------------------------------+
+| Processing step   | EEGLAB function                                              | MNE                                                                         |
++===================+==============================================================+=============================================================================+
+| Get started       | addpath('EEGLAB')                                            | | import mne                                                                |
```

Comment:
I think the most ecologically valid would be just 
```
eeglab;
```

Cause people usually just put EEGLAB in their path IME.

And that could sit parallel to

```
import mne
...
```
etc